app/src/lib/helpers/regex.ts

```diff
@@ -51,3 +51,19 @@ export function getMatches(text: string, re: RegExp): Array<RegExpExecArray> {
 export function escapeRegExp(expression: string) {
   return expression.replace(/[.*+\-?^${}()|[\]\\]/g, '\\$&')
 }
+
+export function getFileFromExceedsError(error: string): string | null {
```

Comment:
Could you add an example of error message in the comments of this function?

Also I think it'd be better if this returned a list of files, instead of an already formatted string.

Reply:
I added an example of the return, and also changed it to return an array of filenames